src/components/PoolingWidget/DefineSpread.tsx

```diff
@@ -35,17 +39,69 @@ export const SpreadInformation: React.FC<SpreadInformationProps> = ({ selectedTo
   )
 }
 
-const DefineSpread: React.FC<DefineSpreadProps> = ({ selectedTokensMap, spread, setSpread }) => {
-  const handleSpreadChange = ({ target: { value } }: React.ChangeEvent<HTMLInputElement>): void => {
-    if (+value < 0 || +value >= 100) return
+const DefineSpread: React.FC<DefineSpreadProps> = ({ selectedTokensMap, spread, setSpread, nextStep }) => {
+  const onChange = useCallback(
+    ({ currentTarget: { value } }: React.SyntheticEvent<HTMLInputElement>): void => {
+      const newValue = value
+        // put a `0` at the start, if starting with `.`
+        .replace(/^(\..*)$/, '0$1')
+        // remove extra decimal places, if present and more than 1
+        .replace(/^(.*\.\d{0,1}).*$/, '$1')
```

Comment:
Small simplification suggestion
```suggestion
        .replace(/^\./, '0$&')
        // remove extra decimal places, if present and more than 1
        .replace(/(\.\d)\d+$/, '$1')
```

The most apparent benefit is that `/^(.*\.\d{0,1}).*$/` always matches even `0.4` and therefore runs the replacements function, even though it literally replaces `0.4 -> 0.4`
`(\.\d)\d+$` doesn't match `0.4` and doesn't run replacement

Reply:
I'm gonna commit this, because it solves a problem with trying to input `0.25`, as per @anxolin experience